lib/arena.rb

```diff
@@ -1,2 +1,45 @@
+# require_relative "..lib/gladiator.rb"
+
 class Arena
+  attr_accessor :name, :gladiators
+
+  def initialize name, gladiators = []
+    @name = name.capitalize!
+    @gladiators = gladiators
+  end
+
+  def add_gladiator new_gladiator_name
+    if @gladiators.count ==2
+      puts "You cannot have more than two Gladiators in the arena"
+    else
+      @gladiators << new_gladiator_name
+      puts "#{@gladiators.count}"
+    end
+  end
+
+  def fight
+    if @gladiators.count == 2
+      @gladiatorsS
+      @gladiators.reverse!
+
```

Comment:
maybe start with one condition. if first gladiator has Trident, and second has spear, who wins?